src/color.rs

```diff
@@ -91,6 +149,41 @@ impl Colors {
         }
     }
 
+    fn get_indicator_from_elem(&self, elem: &Elem) -> Option<Indicator> {
+        let indicator_string = match elem {
+            Elem::File { exec, uid } => {
+                if *uid {
```

Comment:
You can simplify this part by using a match: 
```suggestion
            Elem::File { exec, uid } => match (exec, uid) {
                (_, true) => None,
                (true, false) => Some("ex"),
                (false, false) => Some("fi"),
            },
```